ng-deploy/src/ng-deploy/index.ts

```diff
@@ -1,12 +1,44 @@
 import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
 // import {addPackageJsonDependency,  NodeDependencyType} from 'schematics-utilities';
 import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+import { from } from 'rxjs';
+import { listProjects } from '../builders/actions/init';
+
+import * as inquirer from 'inquirer';
+
+const firebaseJson = {
+  hosting: {
+    public: 'dist/',
+    ignore: ['firebase.json', '**/.*', '**/node_modules/**'],
+    rewrites: [
+      {
+        source: '**',
+        destination: '/index.html'
+      }
+    ]
+  }
+};
+
+const firebaserc = (project: string) => ({
+  projects: {
+    default: project
```

Comment:
the primary reason we called firebase init instead of generating the files was to get the project picker, what would be a good alternative way to pick the project?

Reply:
It's because we need to create the files using schematics. If we invoke `firebase.init()` we'll workaround schematics. As result we need to reimplement the project picker, but I'd prefer to do this instead of not letting schematics run dry run/rollback, etc.